pkg/printer/json_test.go

```diff
@@ -0,0 +1,37 @@
+package printer
+
+import (
+	"testing"
+
+	"github.com/doitintl/kube-no-trouble/pkg/judge"
+)
+
+var findingsJsonTesting []judge.Result = []judge.Result{
+	{
+		Name:        "testName1",
+		Kind:        "testKind1",
+		Namespace:   "testNamespace1",
+		ApiVersion:  "v1",
+		RuleSet:     "testRuleset1",
+		ReplaceWith: "testReplaceWith1",
+		Since:       "testSince1",
+	},
+	{
+		Name:        "testName2",
+		Kind:        "testKind2",
+		Namespace:   "testNamespace2",
+		ApiVersion:  "v1",
+		RuleSet:     "testRuleset2",
+		ReplaceWith: "testReplaceWith2",
+		Since:       "testSince2",
+	},
+}
+
+func TestPrintJson(t *testing.T) {
+	printer := &JSONPrinter{}
+	_, err := printer.populateOutput(findingsJsonTesting)
+
+	if err != nil {
```

Comment:
I'm wondering, could we test a bit more the expected output, than just that we don't return an error?